apis/fluent-en_GB/atrium-api-fluent-en_GB-jvm/src/main/kotlin/ch/tutteli/atrium/api/fluent/en_GB/pathAssertions.kt

```diff
@@ -286,6 +286,26 @@ fun <T : Path> Expect<T>.isRegularFile(): Expect<T> =
 fun <T : Path> Expect<T>.isDirectory(): Expect<T> =
     _logicAppend { isDirectory() }
 
+/**
+ * Expects that the subject of the assertion (a [Path]) is a relative path;
+ * meaning that the [Path] specified in this instance does not start at the file system root.
+ *
+ * This assertion _resolves_ symbolic links.
+ * Therefore, if a symbolic link exists at the location the subject points to, search will continue
+ * at the location the link points at.
+ *
+ * This assertion is not atomic with respect to concurrent file system operations on the paths the assertions works on.
+ * Its result, in particular its extended explanations, may be wrong if such concurrent file system operations
+ * take place.
```

Comment:
We should remove this paragraph since—as discussed in the main thread—the `isRelative` check does not need to access the file system.